### PR TITLE
Use prettier

### PR DESCRIPTION
**Pull request by SimenB**

### CODE REVIEW

**Comment by SimenB** [comment]
When prettier/prettier#2398 lands, it'll be even better

File: src/component/app.jsx [hunk 1]
```diff
@@ -157,27 +250,56 @@ export default class App extends Component {
                                 <FooterSection type="middle">
                                     <FooterDropDownSection title="Menu">
                                         <FooterLinkList>
-                                            {createListItem('/features', 'Feature Toggles')}
-                                            {createListItem('/strategies', 'Strategies')}
-                                            {createListItem('/history', 'Event History')}
-                                            {createListItem('/archive', 'Archived Toggles')}
-                                            {createListItem('/applications', 'Applications')}
+                                            {createListItem(
```
Comment (ivarconr) [inline src/component/app.jsx]: Not everything is automatically more readable..

File: src/component/feature/view-component.jsx [hunk 2]
```diff
@@ -66,44 +71,97 @@ export default class ViewFeatureToggleComponent extends React.Component {
...
                 hashHistory.push('/features');
             }
         };
 
         return (
-            <Card shadow={0} className={commonStyles.fullwidth} style={{ overflow: 'visible' }}>
-                <CardTitle style={{ paddingTop: '24px', wordBreak: 'break-all' }}>{featureToggle.name}</CardTitle>
-                <CardText>{featureToggle.description}</CardText>
-                <CardActions border style={{ display: 'flex', alignItems: 'center', justifyContent: 'space-between' }}>
+            <Card
```
Comment (ivarconr) [inline src/component/feature/view-component.jsx]: this looks stupid

Comment (ivarconr) [inline src/component/feature/view-component.jsx]: nvm, was just the whitepsace not showing in the diff.

File: src/component/app.jsx (hunk 1 above)
Comment (SimenB) [inline src/component/app.jsx]: I agree, but this component could be split up into smaller ones, which would decrease the indentation, which would keep it on a single line

Comment (SimenB) [inline src/component/app.jsx]: This render function is 200 lines, that's pretty crazy

**Comment by SimenB** [comment]
Updated to 1.6 beta, and the JSX looks quite a bit better, IMO 🙂 Waiting for full release, then I'll merge

**Comment by SimenB** [comment]
Aaaand, it's out! 🎉

**Comment by ivarconr** [comment]
:fire: Thanks!

**Pull request merged**